Bottom: db7f7901d64ab28c2a0280de3bb0453ca753f97f
Top:    c0fe15fa0457ed208c7be34cd0b7fb2ac37c06e5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 15:17:52 +0100

Remove SWAP_IDENT, pass it as an argument where needed

Magic state is bad ;)


---

diff --git a/src/draw.c b/src/draw.c
index af57288..45c2ce9 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1921,8 +1921,6 @@ hid_expose_callback (HID * hid, BoxType * region, void *item)
 
   Gathering = false;
 
-  /*printf("\033[32mhid_expose_callback, s=%p %d\033[0m\n", &(SWAP_IDENT), SWAP_IDENT); */
-
   hid->set_color (Output.pmGC, "erase");
   hid->set_color (Output.bgGC, "drill");
 
diff --git a/src/macro.h b/src/macro.h
index 19f80b8..95236af 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -34,11 +34,7 @@
 
 /* ---------------------------------------------------------------------------
  * macros to transform coord systems
- * draw.c uses a different definition of TO_SCREEN
  */
-#ifndef	SWAP_IDENT
-#define	SWAP_IDENT			Settings.ShowSolderSide
-#endif
 
 #define	SWAP_SIGN_X(x)		(x)
 #define	SWAP_SIGN_Y(y)		(-(y))
